### PR TITLE
Issue 617 - fixes DLNA RecentAlbum browsing 

### PR DESCRIPTION
**Pull request by akpdw**
Removes an incorrect override in AlbumUpnpProcessor which was preventing
RecentAlbumUpnpProcessor from behaving correctly.

Also adds a correct getAllItemsSize() implementation for RecentAlbumUpnpProcessor.

### CODE REVIEW

File: airsonic-main/src/main/java/org/airsonic/player/service/upnp/AlbumUpnpProcessor.java [hunk 1]
```diff
@@ -61,20 +59,6 @@ public AlbumUpnpProcessor() {
...
-        DIDLContent didl = new DIDLContent();
-
-        List<MusicFolder> allFolders = getDispatchingContentDirectory().getSettingsService().getAllMusicFolders();
-        List<Album> selectedItems = getAlbumDao().getAlphabeticalAlbums((int) firstResult, (int) maxResults, false, true, allFolders);
-        for (Album item : selectedItems) {
-            addItem(didl, item);
-        }
-
-        return createBrowseResult(didl, (int) didl.getCount(), getAllItemsSize());
-    }
```
Comment (muff1nman) [inline airsonic-main/src/main/java/org/airsonic/player/service/upnp/AlbumUpnpProcessor.java]: Just to make sure I understand, the above is just a code cleanup as it does nothing functionally different than the base class, correct?

Comment (akpdw) [reply airsonic-main/src/main/java/org/airsonic/player/service/upnp/AlbumUpnpProcessor.java]: Oh wait--hold off on this. Now I know what happened; I had thought that for some weird reason I'd left in some old un-refactored code which was breaking the RecentAlbumUpnpProcessor. But now that I'm looking more closely I realize that this was actually an optimization that had been put in...which was breaking the RecentAlbumUpnpProcessor.

This fix just removes the optimization, which works but isn't, erm, optimal.  The right thing to do is to move the optimization so it doesn't break its subclasses.

**Comment by akpdw** [comment]
Ok, this restores the optimization in the AlbumProcessor and just re-overrides it in the RecentAlbums subclass.

**Comment by muff1nman** [comment]
Ah okay I understand now. Thanks for making it clearer.

**Pull request merged**